src/test/modelmanager_test.cpp

```diff
@@ -350,6 +350,39 @@ TEST(ModelManager, ConfigReloadingWithWrongInputName) {
     ASSERT_EQ(status, ovms::StatusCode::CONFIG_SHAPE_IS_NOT_IN_NETWORK);
 }
 
+TEST(ModelManager, ConfigReloadingWithTwoModelsWithTheSameName) {
+    const char* configWithTwoSameNames = R"({
+   "model_config_list": [
+    {
+      "config": {
+        "name": "same_name",
+        "base_path": "/tmp/models/dummy1"
+      }
+    },
+    {
+      "config": {
+        "name": "same_name",
+        "base_path": "/tmp/models/dummy2"
+      }
+    }]})";
+    std::filesystem::create_directories(model_1_path);
+    std::filesystem::create_directories(model_2_path);
+    std::string fileToReload = "/tmp/ovms_config_file2.json";
+    createConfigFileWithContent(configWithTwoSameNames, fileToReload);
+    modelMock = std::make_shared<MockModel>();
+    MockModelManager manager;
+
+    EXPECT_CALL(*modelMock, addVersion(_))
```

Comment:
Here you expect StatusCode::OK one time, is it possible to expect error status the second time when name is duplicated?

Reply:
I think no, because when name is duplicated, it just skips model without calling reloadModelWithVersions() and doesn't return anything.